jsettlers.logic/src/main/java/jsettlers/input/GuiInterface.java

```diff
@@ -107,6 +110,14 @@ public GuiInterface(IMapInterfaceConnector connector, IGameClock clock, ITaskSch
 		this.multiplayer = multiplayer;
 		this.constructionMarksCalculator = new ConstructionMarksThread(grid.getConstructionMarksGrid(), clock, player);
 
+		this.refreshSelectionTimer = new Timer("refreshSelectionTimer");
```

Comment:
Please do not use a plain timer for everyting that needs updating.

Use `UIUpdater` instead.